errai-data-binding/src/main/java/org/jboss/errai/databinding/rebind/BindableProxyGenerator.java

```diff
@@ -238,6 +239,10 @@ private void generateGetter(final ClassStructureBuilder<?> classBuilder, final S
       final BlockBuilder<?> getMethod) {
 
     final MetaMethod getterMethod = bindable.getBeanDescriptor().getReadMethodForProperty(property);
+    if(isToBeIgnored(getterMethod)){
```

Comment:
I think it would be better to perform this check at the call-site of `generateGetter`. The issue with the logic currently is that it is now possible to have a model at run-time that has a generated setter but no generated getter. You can see in `generateSetter` that the previous logic ensured that a setter was only generated if a getter method existed.